web/package.json

```diff
@@ -94,9 +96,8 @@
     "@types/jest": "^29.0.0",
     "@types/recharts": "^1.8.23",
     "cypress": "^12.0.0",
-    "eslint": "^8.12.0",
     "eslint-plugin-react-hooks": "^4.4.0",
-    "react-scripts": "^5.0.1",
+    "react-scripts": "5.0.1",
```

Comment:
Do we need an exact version of `react-scripts` for `craco`?

Reply:
Not with yarn!